Gemfile

```diff
@@ -2,3 +2,5 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in draft_punk.gemspec
 gemspec
+
+gem 'amoeba', github: 'amoeba-rb/amoeba', branch: 'master' # Since published versions of this gem not supporting rails 5
```

Comment:
I think, only development dependencies go into this file. This probably won't be picked up by the main app.

Reply:
I don't think so, I was looking for the most appropriate article, but this is the only thing I could find about using Gemfile deps in gemspec:
https://stackoverflow.com/a/19363523

Reply:
I haven't found a good prooflink too. But I remember dealing with this at some point. It was a long time ago though. So the best way is to just try installing it in the main app and see what happens.

Reply:
That's a bad sign. If so, we probably will have to fork amoeba gem as well. I hoped to avoid this.
I'll return to this a bit later after finish out other gems issues.

Reply:
And the good catch guys @astyagun @korki , thank you.

Reply:
Does this gem do lots of stuff for us? Alternatively we could try searching for ways to replace it, maybe there's some easily usable alternative? A replacement would be especially attractive, if it didn't require having our own fork and supported Rails 5.1 and 5.2, unlike this gem.

Reply:
1. I'd first check if it works or not before going into alternatives
2. As it's a dependency of a dependency you can always put amoeba into FuseTube gemfile and on bundling and running it will just work, so no need of forking and maintaining our own versions of public gems.

Reply:
Agree, if it just works, then it's better to postpone search for alternatives until the next Rails upgrade (5.1). But if making this gem work takes considerable amounts of time now, then it depends on what we want to optimize for:
* If for sooner Rails 5.0.7 upgrade delivery, then this gem should probably be used anyway
* If for sooner Rails 5.2 upgrade delivery, then maybe we should start considering alternatives sooner and save time, that would've been required to make this gem work now

But it will probably just work after including the `amoeba` into FuseTube `Gemfile` anyway, so never mind 🙂